src/Sql/Sql/Database/Cmdlet/RevalidateAzureSqlDatabaseTransparentDataEncryptionProtector.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.Azure.Commands.Sql.Database.Model;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.Database.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet for Revalidate Azure Sql Database encryption protector
+    /// </summary>
+    [Cmdlet("Invoke", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlDatabaseTransparentDataEncryptionProtectorRevalidation", SupportsShouldProcess = true), OutputType(typeof(AzureSqlDatabaseModel))]
+    public class RevalidateAzureSqlDatabaseTransparentDataEncryptionProtector : AzureSqlDatabaseCmdletBase<AzureSqlDatabaseModel>
+    {
+        /// <summary>
+        /// Gets or sets the name of the database to revalidate.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            HelpMessage = "The name of the Azure SQL Database to revalidate.")]
+        [Alias("Name")]
+        [ValidateNotNullOrEmpty]
+        public string DatabaseName { get; set; }
+
+        /// <summary>
+        /// Defines whether it is ok to skip the revalidate Transparent Data Encryption protector request confirmation
+        /// </summary>
+        [Parameter(HelpMessage = "Skip confirmation message for performing the action")]
+        public SwitchParameter Force { get; set; }
```

Comment:
Per my knowledge, validating doesn't change data/resource am I correct? Force is needed if we wan to skip confirmation to change resource.  Please check all cmdlets.